docs/install-unix.sh

```
@@ -0,0 +1,101 @@
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+echo -e "${BLUE}Installing Gitoko - Git Cherry-Picking Tool${NC}"
+
+# Detect OS and architecture
+OS="$(uname -s)"
+ARCH="$(uname -m)"
+
+# Set download URL based on latest release
+REPO="github.com/usman1100/gitoko"
```

Shellcheck (0.10.0)

[warning] REPO appears unused. Verify use (or export if used externally).

(SC2034)